ui/sse_server.py

```
@@ -0,0 +1,362 @@
+
+                # Stream live events via polling
+                last_event_count = len(parent._queue.get_buffered_events())
+                last_heartbeat = time.time()
+                heartbeat_interval = 5.0  # Send comment every 5s to keep connection alive
+
+                while parent._running:
+                    events = parent._queue.get_buffered_events()
+                    if len(events) > last_event_count:
+                        for event_json in events[last_event_count:]:
+                            if not self._send_event(event_json):
+                                print(f"[DSS SSE] Client disconnected")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)